diracos/scriptTemplates/bundle_diracos_script_tpl.sh

```diff
@@ -22,8 +22,12 @@ mkdir $DIRACOS
 cd $DIRACOS
 for i in $PKG_URLS; do curl -L $i | rpm2cpio | cpio -dvim; done
 
+# We need to copy the python modules using rsync
+# because some directory are overwriten by files
+yum install -y rsync
+
 echo "Copying python modules"
-cp -r /tmp/pipDirac/lib/* $DIRACOS/usr/lib64/
+rsync -zvr /tmp/pipDirac/lib/ $DIRACOS/usr/lib64/
 cp -r /tmp/pipDirac/bin/ $DIRACOS/usr/
```

Comment:
Maybe rsync both, or is there any reason for not doing it?

Reply:
it's not quite the same operation:
* first one takes all what is under `/tmp/pipDirac/lib` and add its content to `$DIRACOS/usr/lib64/`
* second takes the whole `/tmp/pipDirac/bin` directory and put it as such in `$DIRACOS/usr`

Not sure the second one can even be done with rsync ? I've got to admit I always had a hard time with rsync :D But if you prefer, and have the line ready, I don't mind changing :)